documentation/book/faq.adoc

```diff
@@ -1,9 +1,52 @@
-[appendix]
-== Frequently Asked Questions
 
-=== Cluster Operator
+= Frequently Asked Questions
 
-==== Log contains warnings about failing to acquire lock
+== Cluster Operator
+
+=== Why does {ProductName} cluster admin privileges to install?
+
+When installing {ProductName} you need to have the ability to create Custom Resource Definitions (CRDs). 
+These are the resources that tell {PlatformProductName} about the {ProductName}-specific resources, such as `Kafka`, `KafkaConnect` and so on.
+Because CRDs are a cluster-scoped resource (rather than being scoped to a particular {PlatformProductName} namespace) typically cluster admin priviledges are required to be able to do this.
+
+Installation of {ProductName} also requires the ability to create some `ClusterRoles` and `ClusterRoleBindings`, similarly to CRDs these are cluster-scoped resources and so, again, typically cluster admin privileges are required to be able to do this.
+
+The cluster administrator can inspect all the resources being installed (in the `/install/` directory) to assure themselves that the `ClusterRoles` do not grant unnecessary privileges. For more information about why the Cluster Operator installation resources grant the ability to create `ClusterRoleBindings` see the following question.
+
+After installation, the Cluster Operator will run as a normal workload: Since it is just a regular `Deployment` a non-admin user with privileges to access the `Deployment` can configure it.
```

Comment:
I think _normal workload_ is a bit confusing. It will run with only the privileges granted by our RBAC roles but not as a cluster-admin. _Normal_ workload might suggest its running under the default SA with the default rights.